index.bs

```diff
@@ -203,18 +203,26 @@ for tracking sites is cleared after 24 hours, unless the user has interacted
 with the site in the first-party context in the last 45 days.
 
 <h4 id="mitigations-brave">Brave</h4>
-Brave uses three list-based approaches to combat [=navigational tracking=].
+Brave uses four list-based approaches to combat [=navigational tracking=].
 
 First, Brave strips query parameters commonly used for [=navigational tracking=]
 from URLs on navigation. This list is maintained by Brave.
 
-Second, in the non-default, "aggressive blocking" configuration, Brave uses
+Second, by default, when i) the user is about to visit a list-identified
+bounce-tracking URL, and ii) the current profile does not contain any cookies
+or DOM state for that site, Brave will create a new, "ephemeral", empty storage
+area for the site. This storage area persists as long as the user has
+any top-level frames open for the site. As soon as the user has no
+top-level frames for the labeled bounce-tracking site, the ephemeral storage
+area is deleted.
```

Comment:
If a user opens a tracking URL, and while that URL is open they open another tab for a non-tracking URL on the same site, this isn't clear whether the non-tracking tab
1. uses the ephemeral area that's going to be deleted,
2. promotes the ephemeral area to permanent, or 
3. uses a distinct storage area that's permanent and doesn't see any changes made by the tracking tab.

I'm not sure you *need* to be clear in this section of the document. We could just clarify if/when we write the real spec for the feature.

Reply:
> We could just clarify if/when we write the real spec for the feature

Lets go with this because this'll change in the medium term.  Currently there is only one storage area per top level site, so if one site is using the ephemeral one, all other instances of the site will use it.  You can get a non-tracking URL to (temporarily) use the ephemeral one if you open it up in the way you describe.  Depending on our experience with the feature though, and what we're able to upstream, our plans for the feature are to have "navigations scoped" storage areas, so the browser can maintain multiple top-level storage areas for the same site, and persist/use them using something like session cookie logic